R/derive_param_computed.R

```diff
@@ -30,7 +41,28 @@
 #'   derive the new parameter are specified for this parameter or the
 #'   `constant_parameters` parameter.
 #'
-#'   *Permitted Values:* A character vector of `PARAMCD` values
+#'   If observations should be considered which do not have a parameter code,
+#'   e.g., if an SDTM dataset is used, temporary parameter codes can be derived
+#'   by specifying a list of expressions. The name of the element defines the
+#'   temporary parameter code and the expression the condition for selecting the
+#'   records. For example `parameters = exprs(HGHT = VSTESTCD == "HEIGHT")`
+#'   selects the observations with `VSTESTCD == "HEIGHT"` from the input data
+#'   (`dataset` and `dataset_add`), sets `PARAMCD = "HGHT"` for these
+#'   observations, and adds them to the observations to consider.
+#'
+#'   Unnamed elements in the list of expressions are considered as parameter
+#'   codes. For example, `parameters = exprs(WEIGHT, HGHT = VSTESTCD ==
+#'   "HEIGHT")` uses the parameter code `"WEIGHT"` and creates a temporary
+#'   parameter code `"HGHT"`.
+#'
+#'   *Permitted Values:* A character vector of `PARAMCD` values or a list of expressions
+#'
+#' @param analysis_var Analysis variable
```

Comment:
To the point raised by @jeffreyad [here](https://github.com/pharmaverse/admiral/pull/1926#issuecomment-1563542246) `analysis_var` is restricted to a singular value correct?

Reply:
Yes, as it is is the variable to be set, it does not make sense to specify more than one.